config/masterful_machinery/controllers/stellar_neutron_activator.json

```diff
@@ -0,0 +1,32 @@
+{
+    "controllerId": "stellar_neutron_activator",
+    "textureOverride": "kubejs:block/hempcrete",
```

Comment:
Can we not reference the original texture? I don't really like copying mod assets here 🤔

Reply:
Not that I could make work, no. IE puts it in a sub folder and MM couldn't access it :(

Reply:
just had another thought I'll try.

Reply:
Yeah, I was derpin, gave up too quick

Reply:
Nice!